File: DVLD_Project/Form17.cs
using Guna.UI2.WinForms;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace DVLD_Project
{
    public partial class Form17 : Form
    {
        string _NetionalNo = "", _ClassName="";
        int _AppID = 0, _TestsDone=0;
        Guna2DataGridView _dt=new Guna2DataGridView();
        public Form17()
        {
            InitializeComponent();
        }

        public Form17(int AppID,string ClassName,int TestsDone,string NetionalNo)
        {
            InitializeComponent();
            _AppID = AppID;
            _ClassName = ClassName;
            _TestsDone = TestsDone;
            _NetionalNo=NetionalNo;
        }

        private void Form17_Load(object sender, EventArgs e)
        {
            getInformations1.GetInfoNetionalNo(_NetionalNo);
            getAppInfo1.LoadInformations(_AppID,_ClassName,_TestsDone);
        }
    }
}

File: DVLD_Project/Form17.vb
Imports Guna.UI2.WinForms
Imports System
Imports System.Windows.Forms

Namespace DVLD_Project
    Public Partial Class Form17
        Inherits Form
        Private _NetionalNo As String = "", _ClassName As String = ""
        Private _AppID As Integer = 0, _TestsDone As Integer = 0
        Private _dt As Guna2DataGridView = New Guna2DataGridView()
        Public Sub New()
            InitializeComponent()
        End Sub

        Public Sub New(AppID As Integer, ClassName As String, TestsDone As Integer, NetionalNo As String)
            InitializeComponent()
            _AppID = AppID
            _ClassName = ClassName
            _TestsDone = TestsDone
            _NetionalNo = NetionalNo
        End Sub

        Private Sub Form17_Load(sender As Object, e As EventArgs)
            getInformations1.GetInfoNetionalNo(_NetionalNo)
            getAppInfo1.LoadInformations(_AppID, _ClassName, _TestsDone)
        End Sub
    End Class
End Namespace
